compose/src/main/java/com/bottlerocketstudios/compose/widgets/CircleAvatarImage.kt

```diff
@@ -0,0 +1,45 @@
+package com.bottlerocketstudios.compose.widgets
+
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import coil.compose.AsyncImage
+import com.bottlerocketstudios.compose.R
+import com.bottlerocketstudios.compose.resources.Dimens
+import com.bottlerocketstudios.compose.snippets.snippetDetails.returnMockSnippetDetails
+import com.bottlerocketstudios.compose.util.Preview
+
+@Composable
+fun CircleAvatarImage(
```

Comment:
This could be a candidate to move into LaunchPad library,

Reply:
@BottleRocket-Colin I like the idea but think that before doing that it would be nice to make some things adjustable such as:

* border size (dps)
* border color?
* contentDescription?

Reply:
Indeed

Reply:
@br-jared-stockton that's a great idea. I've added those in my update.